Example/ToolTableViewCell.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  PlanGrid
+//  https://www.plangrid.com
+//  https://medium.com/plangrid-technology
+//
+//  Documentation
+//  https://plangrid.github.io/ReactiveLists
+//
+//  GitHub
+//  https://github.com/plangrid/ReactiveLists
+//
+//  License
+//  Copyright © 2018-present PlanGrid, Inc.
+//  Released under an MIT license: https://opensource.org/licenses/MIT
+// 
+
+import UIKit
+
+final class ToolTableViewCell: UITableViewCell {
+
+}
```

Comment:
one "opinion" encoded in the model:

we get rid of the idea of "reuseIds" and use the class name for the cell. this means you'll need unique cell class names.

however, i think think we should encourage not just using `UITableViewCell`